### PR TITLE
[urlgrab] improve url log file format

### PR DESCRIPTION
**Pull request by tscherf**
Added timestamp to urllog file.

### CODE REVIEW

File: scripts/urlgrab.pl [hunk 1]
```diff
@@ -7,12 +7,15 @@
...
-$VERSION = "0.4";
+$VERSION = "0.5";
 %IRSSI = (
-    authors     => 'David Leadbeater',
-    contact     => 'dgl@dgl.cx',
+    original_authors     => 'David Leadbeater',
+    original_authorscontact     => 'dgl@dgl.cx',
+    original    =>  'Thorsten Scherf',
+    original    =>  'tscherf@redhat.com',
     name        => 'urlgrab',
```
Comment (ailin-nemui) [inline scripts/urlgrab.pl:19]: just add yourself to the authors list with a `,`, don't add any weirdness here

File: scripts/urlgrab.pl [hunk 2]
```diff
@@ -55,11 +58,12 @@ sub find_url {
 }
 
 sub url_log{
+   my $dt3 = DateTime->now(time_zone => 'local');
```
Comment (ailin-nemui) [inline scripts/urlgrab.pl:61]: DateTime is a heavy dependency, maybe you could port it to Time::Piece instead?

**Comment by ailin-nemui** [comment]
@dgl

File: scripts/urlgrab.pl [hunk 3]
```diff
@@ -55,11 +56,12 @@ sub find_url {
 }
 
 sub url_log{
+   my $t = localtime;
    my($where,$channel,$url) = @_;
    return if lc $url eq lc $lasturl; # a tiny bit of protection from spam/flood
    $lasturl = $url;
    open(URLLOG, ">>", $file) or return;
-   print URLLOG time." $where $channel $lasturl\n";
+   print URLLOG $t->datetime . " $channel $where $lasturl\n";
```
Comment (dgl) [inline scripts/urlgrab.pl:64]: I guess I'm probably the only person who has anything to read this format, easy enough to deal with datetime changing and probably better.

Could we not swap `$channel` and `$where` randomly though, please.

**Comment by tscherf** [comment]
@dgl

**Pull request merged**